### ISSUE TITLE
AttributeError at /django-admin/fobi/formhandler/

### ISSUE THREAD

**Issue by max-pfeiffer** [barseghyanartur/django-fobi#55]
Using Django 1.10 and Python 3.4.

```
[12/Jan/2017 17:47:12] ERROR [django.request:124] Internal Server Error: /django-admin/fobi/formhandler/
Traceback (most recent call last):
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/db/models/options.py", line 617, in get_field
    return self.fields_map[field_name]
KeyError: 'plugin_uid_admin'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/contrib/admin/utils.py", line 274, in lookup_field
    f = _get_non_gfk_field(opts, name)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/contrib/admin/utils.py", line 308, in _get_non_gfk_field
    field = opts.get_field(name)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/db/models/options.py", line 619, in get_field
    raise FieldDoesNotExist('%s has no field named %r' % (self.object_name, field_name))
django.core.exceptions.FieldDoesNotExist: FormHandler has no field named 'plugin_uid_admin'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/core/handlers/exception.py", line 39, in inner
    response = get_response(request)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/core/handlers/base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/core/handlers/base.py", line 215, in _get_response
    response = response.render()
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/response.py", line 109, in render
    self.content = self.rendered_content
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/response.py", line 86, in rendered_content
    content = template.render(context, self._request)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/backends/django.py", line 66, in render
    return self.template.render(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/base.py", line 208, in render
    return self._render(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/loader_tags.py", line 174, in render
    return compiled_parent._render(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/loader_tags.py", line 174, in render
    return compiled_parent._render(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/loader_tags.py", line 70, in render
    result = block.nodelist.render(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/loader_tags.py", line 70, in render
    result = block.nodelist.render(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/template/library.py", line 225, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/contrib/admin/templatetags/admin_list.py", line 339, in result_list
    'results': list(results(cl))}
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/contrib/admin/templatetags/admin_list.py", line 315, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/contrib/admin/templatetags/admin_list.py", line 306, in __init__
    super(ResultList, self).__init__(*items)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/contrib/admin/templatetags/admin_list.py", line 217, in items_for_result
    f, attr, value = lookup_field(field_name, result, cl.model_admin)
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/django/contrib/admin/utils.py", line 290, in lookup_field
    value = attr()
  File "/Users/max/python_environments/hapimag_intranet/lib/python3.4/site-packages/fobi/models.py", line 126, in plugin_uid_admin
    return self.__unicode__()
AttributeError: 'FormHandler' object has no attribute '__unicode__'
[12/Jan/2017 17:47:12] ERROR [django.server:131] "GET /django-admin/fobi/formhandler/ HTTP/1.1" 500 350271
```

**Comment by barseghyanartur** [comment]
@max-pfeiffer: Regarding all django-admin related issues, you're not supposed to use Django fobi using classic django-admin. They are still bugs, though, to be fixed. Thanks for bringing it all up.

**Comment by barseghyanartur** [comment]
@max-pfeiffer: Fixed in 0.10.4.
